### PR TITLE
Updating docs to show support for Python 3.6 in Windows. (#12687)

### PR DESCRIPTION
**Pull request by av8ramit**
Cherrypicking into r1.3 for docs gen.

### CODE REVIEW

File: tensorflow/docs_src/install/install_windows.md [hunk 1]
```diff
@@ -71,12 +71,14 @@ Use that package at your own risk.
 
 ## Installing with native pip
 
-If the following version of Python is not installed on your machine,
+If one of the following versions of Python is not installed on your machine,
```
Comment (caisq) [inline tensorflow/docs_src/install/install_windows.md]: nit: one --> neither

Comment (caisq) [inline tensorflow/docs_src/install/install_windows.md]: nit: is not --> is

Comment (av8ramit) [reply tensorflow/docs_src/install/install_windows.md]: Done.

Comment (av8ramit) [reply tensorflow/docs_src/install/install_windows.md]: Done.

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Review by martinwicke — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

**Pull request merged**